librosa/feature/rhythm.py

```diff
@@ -139,6 +139,7 @@ def tempogram(
         raise ParameterError("win_length must be a positive integer")
 
     ac_window = get_window(window, win_length, fftbins=True)
+    ac_window = ac_window.reshape((-1,1)) # explicit broadcast
 
```

Comment:
This needs to expand out to the full shape of the framed onset envelope, not just one leading dimension.